Resources/Prototypes/Catalog/Fills/Lockers/heads.yml

```diff
@@ -42,8 +44,11 @@
       - id: ClothingHeadHatCaptain
       - id: ClothingNeckCloakCap
       - id: ClothingHandsGlovesCaptain
+      - id: ToyFigurineCaptain
+        prob: 0.15
       - id: ClothingOuterHardsuitCap
       - id: ClothingMaskGasCaptain
+      - id: OxygenTankFilled
```

Comment:
Missing nitrogen. Alternatively, why can't the captain visit engineering for a tank?

Reply:
I did add it, but I was told to remov e it by metal gear sloth